build/configs/artik053/artik053_download.sh

```diff
@@ -22,6 +22,12 @@
 
 # Remember, make is invoked from "os" directory
 
+# Include the configuration file
```

Comment:
If you want to change this, README should be modified together. Please modify [README](https://github.com/Samsung/TizenRT/blob/master/build/configs/artik053/README.md#romfs).

Reply:
Ok.